buildbot/buildbot/worker.py

```diff
@@ -230,7 +256,8 @@ def stage_seashell(db, config):
                     break
             else:
                 raise WorkError(
-                    'No hardware source file found. Expected a file with extension {} and basename not `main`.'.format(C_EXT)
+                    'No hardware source file found.' +
+                    ' Expected a file with extension {} and basename not `main`.'.format(C_EXT)
```

Comment:
Python doesn't require `+` to concatenate string literals (to facilitate this kind of wrapping):

```
>>> ('a'
... 'b')
'ab'
```

So you can just leave that `+` off the previous line. (Similarly in the next diff hunk.)

Reply:
B U T   W H Y ?

Fixed.